core/src/apps/common/cbor.py

```diff
@@ -230,7 +230,7 @@ def encode(value: Value) -> bytes:
     return b"".join(_cbor_encode(value))
 
 
-def encode_chunked(value: Value) -> Iterator[bytes]:
+def encode_streamed(value: Value) -> Iterator[bytes]:
```

Comment:
I decided to leave this function there since it's used in `_hash_tx_body`. As an alternative, I could've simply passed in the `max_chunk_size` parameter also in `_hash_tx_body`, but it seemed weird since it doesn't need chunks of a given size.

Reply:
I see, fair point